### PR TITLE
Changes to publish the 1.1 branch separately

### PR DESCRIPTION
**Pull request by huntc**
Also, improved the reporting of an unauthorised update so that tests are easier to change.

Finally, included bintray publishing info so that project-doc is easily published as a bundle. As such, loading project-doc should now be as easy as:

```
conduct load project-doc
```

### CODE REVIEW

File: test/controllers/ApplicationSpec.scala [hunk 1]
```diff
@@ -91,15 +91,34 @@ class ApplicationSpec  extends WordSpecLike with Matchers with EitherValues {
                                      |  "after": "335aa74f382a36c7a3594a8fec14fdd2ac754cb2"
                                      |}""".stripMargin))
 
-      status(call(application.update(), request1)) shouldBe OK
+      contentAsString(call(application.update(), request1)) shouldBe "Site update requested"
```
Comment (fsat) [inline test/controllers/ApplicationSpec.scala:94]: @huntc - don't you still want to assert the HTTP status code?

File: test/controllers/ApplicationSpec.scala [hunk 2]
```diff
@@ -108,9 +127,10 @@ class ApplicationSpec  extends WordSpecLike with Matchers with EitherValues {
                                      |  "after": "335aa74f382a36c7a3594a8fec14fdd2ac754cb2"
                                      |}""".stripMargin))
 
-      status(call(application.update(), request2)) shouldBe OK
+      contentAsString(call(application.update(), request3)) shouldBe "Site update requested"
```
Comment (fsat) [inline test/controllers/ApplicationSpec.scala:130]: @huntc - don't you still want to assert the HTTP status code?

**Comment by fsat** [comment]
@huntc - do you intend to keep two separate commits or should they be squashed?

**Comment by fsat** [comment]
@huntc - few questions, LGTM otherwise.

File: test/controllers/ApplicationSpec.scala (hunk 2 above)
Comment (huntc) [inline test/controllers/ApplicationSpec.scala:130]: I could do that as well - I'm not sure it is necessary though - you will only get that text given an OK response. I felt that it was more useful to check the text here given that a unauthorised attempt will yield response text that is useful to update the tests with i.e. the SHA1.

**Comment by huntc** [comment]
> @huntc - do you intend to keep two separate commits or should they be squashed?

Deliberately two distinct commits as they address separate concerns.

**Pull request merged**